src/storm/solver/MinMaxLinearEquationSolver.cpp

```diff
@@ -263,10 +282,11 @@ template class MinMaxLinearEquationSolver<double>;
 template class MinMaxLinearEquationSolverFactory<double>;
 template class GeneralMinMaxLinearEquationSolverFactory<double>;
 
-#ifdef STORM_HAVE_CARL
 template class MinMaxLinearEquationSolver<storm::RationalNumber>;
 template class MinMaxLinearEquationSolverFactory<storm::RationalNumber>;
 template class GeneralMinMaxLinearEquationSolverFactory<storm::RationalNumber>;
-#endif
-}  // namespace solver
-}  // namespace storm
+
+template class MinMaxLinearEquationSolver<storm::Interval, double>;  // Unclear whether this semantically makes sense.
```

Comment:
I'm not sure what the comment is referring to?

Reply:
Ah, this is indeed confusing.

I think my concern at the time of writing that comment was that in general, not all min/max linear equations may have solutions of this type. I think you once rightfully pointed out, that we are pretending a generality wiht the min/max solvers that may not be adequate. I think I should remove the comment and we should have that discussion outside of the code